graph/src/blockchain/types.rs

```diff
@@ -28,8 +28,8 @@ impl BlockHash {
         hex::encode(&self.0)
     }
 
-    pub fn zero() -> Self {
-        Self::from(H256::zero())
+    pub fn is_zero(&self) -> bool {
+        self.0.iter().all(|item| *item == 0)
```

Comment:
https://docs.rs/primitive-types/0.12.1/primitive_types/struct.H256.html#method.is_zero I think this should work?

Reply:
Sorry, I don't know if I get it. `self.0` is a `Box[u8]` and not a H256.
They implemented something [similar](https://github.com/paritytech/parity-common/blob/7a9cc3b03c5311c24623c66fe8a750903cf34484/fixed-hash/src/hash.rs#L206) for H256, but maybe there is an easier way.

Reply:
ah no, you're right then, I assumed it was an H256. The implementation is similar for H256, I just thought it was an H256 so it would have been good to re-use